recipes/OpenFOAM-Infiniband-IntelMPI/README.md

```diff
@@ -16,6 +16,7 @@ The pool configuration should enable the following properties:
 `platform_image` with IB/RDMA as
 [supported by Batch Shipyard](../../docs/25-batch-shipyard-platform-image-support.md).
 * `inter_node_communication_enabled` must be set to `true`
+* `per_job_auto_scratch` must be set to `true`
```

Comment:
Append: A job autoscratch is needed to share a common input data set between the nodes.